lgc/patch/Gfx9ConfigBuilder.cpp

```diff
@@ -728,6 +736,71 @@ void ConfigBuilder::buildPipelineNggVsTsGsFsRegConfig() {
   appendConfig(config);
 }
 
+// =====================================================================================================================
+// Builds register configuration for graphics pipeline (Mesh-FS).
+void ConfigBuilder::buildPipelineMeshFsConfig() {
+  GfxIpVersion gfxIp = m_pipelineState->getTargetInfo().getGfxIpVersion();
+  assert(gfxIp >= GfxIpVersion({10, 3})); // Must be GFX10.3+
+
+  PipelineMeshFsRegConfig config(gfxIp);
+
+  addApiHwShaderMapping(ShaderStageMesh, Util::Abi::HwShaderGs);
+  addApiHwShaderMapping(ShaderStageFragment, Util::Abi::HwShaderPs);
+
+  setPipelineType(Util::Abi::PipelineType::Mesh);
+
+  {
+    // Must contain mesh shader
+    assert(m_pipelineState->hasShaderStage(ShaderStageMesh));
+    buildMeshRegConfig<PipelineMeshFsRegConfig>(ShaderStageMesh, &config);
+  }
```

Comment:
Do we need this code block? I'd guess that this might be to accommodate some internal code that was cut out during sanitization, but wanted to confirm.

Reply:
also below